examples/iso_dh/DY.Example.DH.Protocol.Stateful.Proof.fst

```diff
@@ -116,33 +116,21 @@ instance dh_protocol_invs: protocol_invariants = {
 
 /// Lemmas that the global state predicate contains all the local ones
 
-val all_sessions_has_all_sessions: unit -> Lemma (norm [delta_only [`%all_sessions; `%for_allP]; iota; zeta] (for_allP (has_local_bytes_state_predicate dh_protocol_invs) all_sessions))
-let all_sessions_has_all_sessions () =
+val all_sessions_has_all_sessions: squash (norm [delta_only [`%all_sessions; `%for_allP]; iota; zeta] (for_allP (has_local_bytes_state_predicate dh_protocol_invs) all_sessions))
```

Comment:
Isn't it possible to parameterize this function with the `dh_protocol_invs` and `all_sessions` variables and move the function into the API?

Reply:
My intuition tells me we can't do that, but that's a good point, I will test that!